src/lib/components/Swap/index.tsx

```diff
@@ -59,7 +59,7 @@ export default function Swap(props: SwapProps) {
   return (
     <SwapPropValidator {...props}>
       <SwapInfoUpdater />
-      <Header logo title={<Trans>Swap</Trans>}>
+      <Header title={<Trans>Swap</Trans>}>
```

Comment:
Instead of removing the `logo` and including it in the `Output`, make a new `Footer` component that you can put _after_ `Output`. The footer has nothing to do with the output, so it should be a separate component.